COORDINATION.md

```diff
@@ -0,0 +1,92 @@
+# Project Coordination Guidelines
+
+Arguably, communication is our primary concern throughout the entire process of
+designing, implementing, documenting, testing, and disseminating our programs.
+But, without question, collaboration on software development calls for a lot of
+communication.
+
+This document records our shared (always evolving) understanding of the key ways
+we communicate in furtherance of our collaboration.
+
+## Elements of our project coordination
+
+We currently use GitHub as the main platform for sharing the communications
+that let us coordinate our work. GitHub has a number of different features for
+enabling cooperative work, but it's not necessarily obvious what the scope or
+intended meaning of each feature is. The following table is used to align our
+shared understanding of how to use and interpret the main features:
+
+| Feature       | Scope                                                   | Communicates...        |
+|---------------|---------------------------------------------------------|------------------------|
+| issue         | a distinct problem or task                              | what to work on        |
+| pull request  | (part of) a solution or fulfillment of a task           | the ongoing work       |
+| milestone     | a distinct feature or a closely related set of problems | what work is towards   |
+| label         | a more or less vague category/topic/quality             | attributes of the work |
+| project board | concurrent human processes with a common dynamic        | how work is done       |
+| discussion    | free ranging and open discussion                        | whatever               |
+
+(For more on projects vs. milestones, see
+https://stackoverflow.com/a/47542346/1187277)
+
+### Project board
+
+#### Overall workflow reflection
```

Comment:
Just a shoutout ,since we did the initial draft on Hedgedoc so the commit history wasn't tracked: The part of the document from this point on is all @gabrielamafra's work. 

I find it super helpful to have this kind of big picture, philosophical grounding when considering how we manage the project, and this section on the temporal structure of the project is the part of the document I find most helpful, personally.